lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/data/179d_ashrae_90_1_2007.schedules.json

```diff
@@ -2624,4 +2624,4 @@
       ]
     }
   ]
-}
+}
```

Comment:
Our linter for end of file (pre-commit) on bem to surrogate gem will complain about this change.